checker/examples/GradleExamples/GradleJava7Example/build.gradle

```diff
@@ -0,0 +1,60 @@
+version '1.0-SNAPSHOT'
+
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+if (JavaVersion.current().isJava8Compatible()) {
+    throw new GradleException("This example requires Java 7, but you are using Java 8. Please see the Java 8 example in ../GradleJava8Example")
+}
+
+assert JavaVersion.current().isJava7()
```

Comment:
Rather than asserting use of Java 7, could the Gradle instructions check whether Java 7 is being used, and adjust compile.options.compilerArgs and other build parameters in that case?  This would reduce duplication in the code, the examples, and the instructions.  It would also enable users to upgrade from Java 7 to Java 8 without having to change their Maven build files.